src/i18n/translations.ts

```diff
@@ -282,6 +284,8 @@ export default {
         text: 'Annotation',
         modify: 'Edit',
         upload: 'Upload',
+        uploadSuccess: 'The file was uploaded successfully',
+        uploadError: 'Only files of the type .geojson are supported',
```

Comment:
I assume the upload can also fail because of other reasons. Can we change it so something like "The import failed, please note that only .geojson files are supported".

Reply:
Good point! I have adjusted it according to your suggestion.